packages/browser-wallet/src/popup/page-layouts/ExternalRequestLayout/ExternalRequestLayout.scss

```diff
@@ -4,7 +4,6 @@
     background-color: $color-bg;
 
     &__main {
-        padding: rem(10px);
```

Comment:
This could affect a lot of other UI screens. I can see that you manually added the padding to all the other screens.